internal/cli/cmd/cluster/create.go

```diff
@@ -1335,6 +1363,83 @@ func (o *CreateOptions) buildAnnotation(cls *appsv1alpha1.Cluster) {
 	}
 }
 
+func (o *CreateOptions) buildBackupConfig(cls *appsv1alpha1.Cluster) error {
+	// set default value in here
+	// if set in flag, we don't know whether it is set by user or default value
```

Comment:
https://github.com/apecloud/kubeblocks/pull/4924/files#diff-e4af3b44dfbbba4e2a5f96732b98f5009151028b03c55e7fb2b2ab7f9f4ab087R144-R148

This is the method how to check whether flag is set by user or not.